config/default.go

```diff
@@ -51,6 +51,7 @@ var defaultConfig = &Config{
 			ServiceStatus: []string{"passing"},
 			CheckInterval: time.Second,
 			CheckTimeout:  3 * time.Second,
+			CheckScheme:   "http",
```

Comment:
I'm not sure what you mean by this and the change in `registry/consul/register.go`